.github/workflows/restore-build-test.yml

```
@@ -0,0 +1,42 @@
+name: restore-build-test
+
+on:
+  workflow_call:
+    inputs:
+      dotnet-version:
+            type: string
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 12

(indentation)